# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/udev/rules-kvm.patch
# Copyright (C) 2023 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- udev-182/rules/50-udev-default.rules.vanilla	2023-03-07 13:46:02.558038170 +0100
+++ udev-182/rules/50-udev-default.rules	2023-03-07 13:48:44.410030017 +0100
@@ -66,6 +66,9 @@
 # block
 SUBSYSTEM=="block", GROUP="disk"
 
+# kvm
+KERNEL=="kvm", GROUP="kvm"
+
 # floppy
 SUBSYSTEM=="block", KERNEL=="fd[0-9]", GROUP="floppy"
 
